### PR TITLE
Ncnews 10 get articles queries

### PR DESCRIPTION
**Pull request by michaelafm**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -111,6 +111,75 @@ describe("/api/articles", () => {
...
+  test("GET: 200 - sort by column given as the sort_by query and order query", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author&order=ASC")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("author");
+        expect(body.articles[0].author).toBe("butter_bridge");
+      });
+  });
+  test("GET: 200 - returns empty array when given non-existent topic", () => {
```
Comment (callumu21) [inline __tests__/app.test.js:157]: You will need to consider two different cases here:

- Happy path: The topic is in the database but has no associated articles
- Sad path: The topic is invalid and not in the database

**Review by callumu21 — CHANGES_REQUESTED** [review]
Great start! Just need to consider two different test cases and how you can differentiate between them in your model

File: __tests__/app.test.js (hunk 1 above)
Comment (michaelafm) [reply __tests__/app.test.js]: Two distinct tests added for topic with no associated articles and invalid topic

**Comment by Geo-Beard** [comment]
Changes implemented - looks good! Feel free to merge and move on :)

**Pull request merged**